src/main/java/fi/otavanopisto/kuntaapi/server/integrations/casem/CaseMCacheUpdater.java

```diff
@@ -882,6 +894,11 @@ private void updateTreeNode(OrganizationId organizationId, Node parentNode, Node
     identifierRelationController.setParentId(identifier, identifierParentId);
 
     PageId kuntaApiPageId = new PageId(organizationId, KuntaApiConsts.IDENTIFIER_NAME, identifier.getKuntaApiId());
+    
+    if (kuntaApiPageId != null && kuntaApiParentPageId != null) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to "true" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2583)